meteor/client/ui/Settings/Studio/PackageManager.tsx

```diff
@@ -488,6 +488,36 @@ export const StudioPackageManagerSettings = withTranslation()(
 														{t('Base url to the resource (example: http://myserver/folder)')}
 													</span>
 												</label>
+												<label className="field">
+													<LabelActual label={t('Is Immutable')} />
+													<EditAttribute
+														modifiedClassName="bghl"
+														attribute={`packageContainers.${containerId}.container.accessors.${accessorId}.isImmutable`}
+														obj={this.props.studio}
+														type="checkbox"
+														collection={Studios}
+														className="input text-input input-l"
+													></EditAttribute>
+													<span className="text-s dimmed field-hint">
+														{t('When set, resources are considered immutable, ie they will not change')}
+													</span>
+												</label>
+												<label className="field">
+													<LabelActual label={t('Supports HEAD requests')} />
+													<EditAttribute
+														modifiedClassName="bghl"
+														attribute={`packageContainers.${containerId}.container.accessors.${accessorId}.supportHEAD`}
+														obj={this.props.studio}
+														type="checkbox"
+														collection={Studios}
+														className="input text-input input-l"
+													></EditAttribute>
+													<span className="text-s dimmed field-hint">
+														{t(
+															'Set to true if the HTTP server supports HEAD requests. (otherwise GET requests will be used to check availability)'
+														)}
```

Comment:
Any reasonable HTTP server should support HEAD in 2024 - can we switch the default around? To say, "Use GET requests instead of HEAD to check availability"?

`express.js` generates a HEAD handler automatically for GET routes: https://expressjs.com/en/api.html#app.METHOD - I think so does `koa-router`, so I think it's safe to assume that HEAD is supported, _unless_ it isn't, not the other way round.

Reply:
Yeah, that's a good point, that would also be solve a backwards compatibility issue.